#310: improved documentation (#316)
exasol/virtual-schemas: doc/development/developing-sql-dialect/step_by_step_guide_to_writing_your_own_dialect.md

_Athena is based on the Open Source project [Apache Presto](https://prestodb.github.io) which in the own words of the Presto team is a "distributed SQL query engine for Big Data". 
In short it's a cluster of machines digging through large amounts of data stored on a distributed file system._

## Prerequisites

To start developing Virtual Schemas you should know:

* How to use [git](https://git-scm.com/)
* How to use [Maven](https://maven.apache.org/). Check the list of dependencies and plugins that we use in the project at the end of [README](../../../README.md) file.

> * How to setup Maven project in an IDE of your preference. 

Reviewer: ```suggestion
* How to setup a Maven project in the IDE of choice. 
```
